rpcserver.go

```diff
@@ -2936,6 +2939,174 @@ func (s *rpcServer) determineAccountVersion(
 	return account.VersionInitialNoVersion, nil
 }
 
+type ActionFee struct {
```

Comment:
I'd call this something like `ModificationFee`, since Action and Auction look very similar :sweat_smile: